libs/common/src/test-helpers/mock-data-backend/portal-facility-amendment.ts

```diff
@@ -3,11 +3,12 @@ import { ObjectId } from 'mongodb';
 import { AMENDMENT_TYPES, CURRENCY, PORTAL_AMENDMENT_STATUS } from '../../constants';
 import { PortalFacilityAmendment } from '../../types';
 import { PortalFacilityAmendmentUserValues } from '../../types/portal/amendment';
+import { getEpochMs } from '../../helpers';
 
 export const aPortalFacilityAmendmentUserValues = (): PortalFacilityAmendmentUserValues => ({
   changeCoverEndDate: true,
-  coverEndDate: getUnixTime(new Date()),
-  currentCoverEndDate: getUnixTime(new Date()),
+  coverEndDate: getEpochMs(new Date()),
```

Comment:
```suggestion
  coverEndDate: getEpochMs(),
```
as this will default to now.